app/models/import/builder.rb

```diff
@@ -0,0 +1,13 @@
+class Import
+  class Builder
+    def initialize(gqueries, id:, scaling:, **)
+      @gqueries    = gqueries
+      @scenario_id = id
+      @scaling     = scaling
+    end
+
+    def build(response = nil)
```

Comment:
Is there ever a case when you'd call `build` without a response? It seems to be a mandatory argument in some builders, but not in others; I would suggest it should be a mandatory argument for all so that each class has an identical public API.

If the response isn't used anywhere in the method, use the Rubocop convention of prepending an `_` to the name:

```ruby
def build(_response)
  # ...
end
```